conda/lock.py

```diff
@@ -120,7 +121,7 @@ def __init__(self, directory_path, retries=10):
             try:
                 os.makedirs(self.directory_path)
                 log.debug("forced to create %s", self.directory_path)
-            except OSError as e:
+            except OSError as e:  # pragma: no cover
```

Comment:
what is "#pragma no cover "

Reply:
excludes the line/clause from coverage report: https://coverage.readthedocs.io/en/6.3.3/excluding.html